### ISSUE TITLE
Mesos README problems

### ISSUE THREAD

**Issue by siddharthist** [CiscoCloud/microservices-infrastructure#886]
The README for the Mesos role mentions variables, [but they aren't visible in the Github rendering](https://github.com/CiscoCloud/microservices-infrastructure/tree/aeb30e6703b053d889c3326a369adb5429cac506/roles/mesos).

**Comment by BrianHicks** [comment]
Nope, they're RST directives that Github just doesn't support. That's why we've been trying to be diligent about directing people to the [equivalent RTD link](http://microservices-infrastructure.readthedocs.org/en/latest/components/mesos.html).

I suppose we could have some other form of documentation (markdown?) but having the docs site updated for us on every push to master is worth the RST overhead, I think.

**Comment by BrianHicks** [comment]
A little research later, it looks like it would be possible to use [mkdocs](http://www.mkdocs.org/) with readthedocs. This might be worth the pain of changing, since it would mean that people could view our documentation on Github as well as the docs site. We wouldn't get the fancy RST directives, but more people are used to writing Markdown anyway. It would make the most sense to me to do this as part of the microservices-infrastructure -> mantl push.

@stevendborrelli, think the effort would be worth it? I do.

**Comment by sehqlr** [comment]
@siddharthist created hammer.readthedocs.org using mkdocs using same theme as mantl's docs

**Comment by ericsakowski** [comment]
None of the links to :doc:<whatever> work in github either.  RTD seems like it's out of date from master.  Fx if I look at https://github.com/CiscoCloud/mantl/blob/master/docs/getting_started/aws.rst it states:  

"Please note, newer versions of this file do not have "access_key" or "secret_key" lines, we automatically find your AWS credentials from amazon's new "AWS Credentials file" standard."

on RTD at http://microservices-infrastructure.readthedocs.io/en/latest/getting_started/aws.html it tells me to put provider into aws.tf.  

Google brings up github for 'mantl aws'...readthedocs isn't even on the first page of results.  I didn't find RTD until i went to file an issue and found it mentioned in this issue.  Seems like it'd save some confusion to have the docs working in github.

**Comment by siddharthist** [comment]
We should take down microservices-infrastructure.readthedocs.io as soon as possible.

**Comment by BrianHicks** [comment]
Agreed. It's been about 6 months, I've removed the old link. Long live docs.mantl.io!
